R/tar_quarto_files.R

```diff
@@ -60,12 +60,31 @@ tar_quarto_files <- function(path = ".", profile = NULL) {
 tar_quarto_files_document <- function(path) {
   info <- quarto::quarto_inspect(input = path)
   out <- list(sources = path)
+
+  # Collect data about output files.
   for (format in info$formats) {
     out$output <- c(
       out$output,
       file.path(dirname(path), format$pandoc$`output-file`)
     )
   }
+
+  # Collect data about input files.
+  for (myfile in names(info$fileInformation)) {
```

Comment:
Looks like `fileInformation` was recently added somewhere between Quarto CLI 1.3.450 and 1.5.57 (based on my local upgrade just now).

Reply:
I have the impression that the output of `quarto inspect` is everything but easily comprehensible. I hope that they won't change the structure too often.